src/Aspire.Hosting/Utils/HostNameResolver.cs

```diff
@@ -8,7 +8,7 @@ namespace Aspire.Hosting.Utils;
 /// <summary>
 /// Helpers to resolve host names when running in a containers.
 /// </summary>
-public class HostNameResolver
+internal class HostNameResolver
```

Comment:
`internal static class HostNameResolver`

No reason to ever create one of these.